javascript/lib.cpp

```diff
@@ -129,6 +129,7 @@ void CompiledIndex::Load(Napi::CallbackInfo const& ctx) {
         auto result = native_->load(path.c_str());
         if (!result)
             Napi::TypeError::New(ctx.Env(), result.error.release()).ThrowAsJavaScriptException();
+        native_->reserve(ceil2(native_->size()));
```

Comment:
I believe in this context it's cleaner to use `try_reserve`, that return a boolean, and if it's `false` - propagate an `Napi` error.